app/utils/room.js

```diff
@@ -18,19 +18,35 @@ export const capitalize = (s) => {
 	return s.charAt(0).toUpperCase() + s.slice(1);
 };
 
-export const formatDate = date => moment(date).calendar(null, {
-	lastDay: `[${ I18n.t('Yesterday') }]`,
-	sameDay: 'LT',
-	lastWeek: 'dddd',
-	sameElse: 'L'
-});
-
-export const formatDateThreads = date => moment(date).calendar(null, {
-	sameDay: 'LT',
-	lastDay: `[${ I18n.t('Yesterday') }] LT`,
-	lastWeek: 'dddd LT',
-	sameElse: 'LL'
-});
+export const formatDate = (date) => {
```

Comment:
Can we write tests for this?
You can mock `new Date()` making it fixed in time and then compare it against other values.
It'd be awesome to make sure it returns excepted values and it also serves as docs.